now.json

```diff
@@ -1,23 +1,21 @@
 {
   "name": "primer-components",
+  "version": 2,
   "alias": "primer-components.now.sh",
-  "scale": {
-    "sfo": {
-      "min": 1,
-      "max": 1
-    }
-  },
-  "engines": {
-    "node": ">=10"
-  },
   "files": [
     ".babelrc",
     "css.js",
-    "next.config.js",
     "package-lock.json",
-    "pages",
+    "docs",
     "rollup.config.js",
     "src",
     "static"
+  ],
+  "builds": [
+    {
+      "src": "docs/package.json",
+      "use": "@now/static-build",
+      "config": {"distDir": "public"}
+    }
```

Comment:
We should set up a `/components` path prefix so the `primer.style/components` domain continues to work. Here's the documentation for that: https://primer.style/doctocat/usage/deployment#adding-a-primerstyle-path-alias